kani-compiler/src/codegen_cprover_gotoc/context/current_fn.rs

```diff
@@ -3,26 +3,26 @@
 
 use crate::codegen_cprover_gotoc::GotocCtx;
 use cbmc::goto_program::Stmt;
-use rustc_middle::mir::BasicBlock;
-use rustc_middle::mir::Body;
-use rustc_middle::ty::Instance;
-use rustc_middle::ty::PolyFnSig;
+use rustc_middle::mir::{BasicBlock, Body as InternalBody};
+use rustc_middle::ty::{Instance as InternalInstance, PolyFnSig};
+use rustc_smir::rustc_internal;
+use stable_mir::mir::mono::Instance;
+use stable_mir::mir::Body;
+use stable_mir::CrateDef;
 
 /// This structure represents useful data about the function we are currently compiling.
 #[derive(Debug)]
 pub struct CurrentFnCtx<'tcx> {
     /// The GOTO block we are compiling into
     block: Vec<Stmt>,
-    /// The current MIR basic block
-    current_bb: Option<BasicBlock>,
```

Comment:
How do we keep track of `current_bb` now? Is it kept by the `tcx` in stableMIR?

Reply:
We don't need to. That information wasn't used at all